### ISSUE TITLE
Screen reader users cannot accept next edit suggestions from within accessible view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#244651]
1. Enable next edit suggestions
2. Using a screen reader, hear sound for inline suggestion and alert to inspect it in the accessible view
3. Open the accessible view, try to accept the suggestion 
4. 🐛 nothing happens

<img width="807" alt="Image" src="https://github.com/user-attachments/assets/0b59a1e3-1b26-44be-8c2e-23c8ef036d8f" />
